### PR TITLE
LLM: Modify CPU Installation Command for documentation

### PR DESCRIPTION
**Pull request by xiangyuT**
## Description

- Modify CPU Installation Command for documentation
- Add example template example readme for updated cpu installation command

https://xiangyutestdoc.readthedocs.io/en/example_modify_0516/doc/LLM/Overview/install_cpu.html

### CODE REVIEW

**Review by Oscilloscope98 — COMMENTED** [review]
For documentation, should we also update here: https://github.com/intel-analytics/ipex-llm/pull/11042/files#diff-0a501c2c482e5664cedf19d45b541930b7f77cca30af7f89171d26211406fedfL3-L9?

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md [hunk 1]
```diff
@@ -8,11 +8,22 @@ To run these examples with IPEX-LLM, we have some recommended requirements for y
...
 conda create -n llm python=3.11
 conda activate llm
 
-pip install ipex-llm[all] # install ipex-llm with 'all' option
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install ipex-llm with 'all' option
+pip install transformers_stream_generator  # additional package required for Baichuan-13B-Chat to conduct generation
+```
+
+On Windows:
+```powershell
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md:22]: As we recommend using cmd for GPU on Windows, maybe it is better with cmd here?
```
```cmd
```

File: docs/readthedocs/source/doc/LLM/Overview/install_cpu.md [hunk 2]
```diff
@@ -41,11 +51,26 @@ For optimal performance with LLM models using IPEX-LLM optimizations on Intel CP
...
+      .. code-block:: bash
+
+         conda create -n llm python=3.11
+         conda activate llm
+
+         pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install the latest ipex-llm nightly build with 'all' option
+
+   .. tab:: Windows
+
+      .. code-block:: bash
```
Comment (Oscilloscope98) [inline docs/readthedocs/source/doc/LLM/Overview/install_cpu.md:68]: Same here `.. code-block:: cmd`

File: docs/readthedocs/source/doc/LLM/Overview/install_cpu.md [hunk 3]
```diff
@@ -4,8 +4,18 @@
...
+.. tabs::
+
+   .. tab:: Linux
+
+      .. code-block:: bash
+         pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install the latest ipex-llm nightly build with 'all' option
+
+   .. tab:: Windows
+
+      .. code-block:: bash
```
Comment (Oscilloscope98) [inline docs/readthedocs/source/doc/LLM/Overview/install_cpu.md:17]: image

Wrong syntax here: there should be an empty line beteen `.. code-black:: ` and the contents

Besides, maybe `.. code-block:: cmd` is better

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md (hunk 1 above)
Comment (xiangyuT) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md:22]: Do we need to change line 46 ```powershell to ```cmd to align?
image

Comment (Oscilloscope98) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan/README.md:22]: I think we can do that :)

File: docs/readthedocs/source/doc/LLM/Overview/install_cpu.md [hunk 4]
```diff
@@ -4,8 +4,20 @@
...
-```bash
-pip install --pre --upgrade ipex-llm[all] # install the latest ipex-llm nightly build with 'all' option
+```eval_rst	
+.. tabs::
+
+   .. tab:: Linux
+
+      .. code-block:: bash
+
+         pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install the latest ipex-llm nightly build with 'all' option
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/LLM/Overview/install_cpu.md:14]: Maybe we can put `# in front of command`.

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**